### ISSUE TITLE
Node v0.12.2 connections don't close

### ISSUE THREAD

**Issue by NathanielInman** [olalonde/follow-redirects#21]
Connections don't close on node v0.12.2 due to [this line](https://github.com/olalonde/follow-redirects/blame/master/create.js#L43). The node server will eventually stop responding. In a production app I fixed it by commenting the line out.

Maybe putting a conditional for node version would be a better solution?

Thanks!

**Comment by jamestalmage** [comment]
@NathanielInman Does the problem exist in v0.12.2?

**Comment by olalonde** [comment]
Would be nice to have a test case. https://travis-ci.org/olalonde/follow-redirects doesn't seem to pick up that bug with current tests.

**Comment by jamestalmage** [comment]
OK,
I think this applies across all versions.

Temporary workaround is to listen to `end` events instead of `close`

**Comment by jamestalmage** [comment]
@NathanielInman 
I am having trouble reproducing, do you have a code snippet you can share?
I do not see `close` events even from the builtin `http` module

```js
http.get('http://example.com/', function(res) {
  consume(res);
  res.on('close', /* never gets called */);
  res.on('end', /* always gets called */);
});
```

**Comment by jamestalmage** [comment]
@NathanielInman 
@c084e82386da1cbf9af064d49343325dd082291c published as `follow-redirects@0.0.7`. Try it and see if it fixes your issue. I decided to proceed with that commit whether it fixed your issue or not (see #22).

**Even if it does work for you**, I would still really appreciate a minimal reproduction of what caused `0.0.6` to fail, so we can add it to the test suite and avoid breaking changes in the future.
